docs/take-to-prod.md

```diff
@@ -0,0 +1,85 @@
+Taking your .NET for Apache Spark Application to Production
```

Comment:
comments from @suhsteve  "This title somehow makes me feel like after reading this I will know how to make my spark application production ready. Is this the purpose of this document ? Or just to outline different spark-submit scenarios."

I believe the purpose of this doc should tell users how to move their application to production. I am thinking of putting different scenarios along with the instruction on how to move it to production based on these different scenarios. Any suggestions to make this doc more precise and explicit would be really appreciate.